doc/tutorials/application-integration.md

```diff
@@ -30,57 +30,54 @@ additional benefits arise:
-Elektra's infrastructure solving basic issues like getting configuration
-from other parts of the system, update and conflict detection, and
-resolving of the file name. In particular we gain a lot because
+We call the process of making applications aware of other's configuration
+"to elektrify". This tutorial is suited both for new and existing applications.
+
+As first step, locate places where configuration is parsed or generated.
+Afterwards, use Elektra's data structures instead at these locations.
+Before we are going to describe how to do this, we will describe
+some possibilities to keep all advantages your previous configuration 
+system had.
+
+You can keep code you want within Elektra as plugins. This allows your
+application, and other applications participating in Elektra's ecosystem to
+access your configuration. Doing this, the syntax of the configuration file
+stays the same as before. You can keep the same validation as you had before.
+The application profits from Elektra's infrastructure solving basic issues like
+getting configuration from other parts of the system, update and conflict
+detection, and resolving of the file name. In particular we gain a lot because
 every other program can also access the configuration of your software.
 
-If you do not have the code or want to get rid of it, a huge variety of
-[already implemented plugins](/src/plugins)
-can be utilised to extend the functionality of the configuration
-system.  There are plenty of plugins that parse and generate
-configuration files in many formats, do syntactic checks, do
+If you do not have the code or want to get rid of it, you can use a variety of
+[already implemented plugins](/src/plugins) to extend the functionality of the
+configuration system. There are plenty of plugins that parse and generate
+configuration files in different formats, do syntactic checks, do
 notifications (e.g. via dbus), and write out events in their log files.
 
-New applications simply do not have the burden to stay compatible with
-the configuration system they had to before.  So they will prefer
+New applications do not have the burden to stay compatible with
+the configuration system they had to before. So they will prefer
 to use more standard plugins and contribute to make them flawless.
-But they can also use self-written plugins for adding needed behaviour
+But they can also use self-written plugins for adding needed behavior
 or cross-cutting concerns.
 
 To sum up, if a developer wants to **elektrify** software, he or she can
 do that without any need for changes to the outside world regarding the
-format and semantics of the configuration.  But in the interconnected
-world it is only a matter of time until other software also wants to
-access the configuration, and with elektrified software it is possible
+format and semantics of the configuration. In the interconnected
+world it's a matter of time until other software also wants to
```

Comment:
i guess thats a matter of preference. i think it's is a bit easier to read and fine to use unless writing a paper. But grep reveiled that it is is much more common throughout our documents than it's though, so ill stick to that.